### PR TITLE
Update to eslint-config-pretter 8

### PR DESCRIPTION
**Pull request by somewhatabstract**
## Summary:
This updates our deps to the latest and modifies our config to match
the changes to eslint-config-pretty.

Issue: XXX-XXXX

## Test plan:
Added it to render-gateway and checked it worked

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@khanacademy/eslint-config",
-    "version": "0.0.5",
+    "version": "0.1.0",
```
Comment (k4b7) [inline package.json:3]: Big changes!

File: package.json [hunk 2]
```diff
@@ -10,24 +10,24 @@
...
-        "eslint-plugin-graphql": "2.1.1",
-        "eslint-plugin-import": "^2.18.0",
-        "eslint-plugin-jsx-a11y": "^6.1.2",
-        "eslint-plugin-prettier": "3.0.0",
-        "eslint-plugin-react": "7.11.1",
-        "eslint-plugin-react-hooks": "^1.6.1",
-        "eslint-plugin-react-native": "^3.7.0",
-        "eslint-plugin-react-native-animation-linter": "0.1.2"
+        "eslint": "^7.27.0",
+        "eslint-config-prettier": "^8.3.0",
```
Comment (k4b7) [inline package.json:14]: Oh wow... I didn't realize this was so out of date.  Thanks for updating.

File: package.json [hunk 3]
```diff
@@ -10,24 +10,24 @@
...
+        "eslint-plugin-react": "^7.24.0",
+        "eslint-plugin-react-hooks": "^4.2.0",
+        "eslint-plugin-react-native": "^3.11.0",
+        "eslint-plugin-react-native-animation-linter": "^0.1.2"
     },
     "devDependencies": {
         "pre-commit": "^1.2.2",
-        "prettier": "^1.19.1",
-        "pretty-quick": "^2.0.1"
+        "prettier": "^2.3.0",
```
Comment (k4b7) [inline package.json:29]: Good call.  We should probably update webapp to 2.x at some point.

**Review by k4b7 — APPROVED** [review]
LGTM.  Thanks for updating the deps.

File: package.json (hunk 3 above)
Comment (somewhatabstract) [reply package.json:29]: Yeah, at some point. :)

**Pull request merged**